<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main_wd</class>
 <widget class="QDialog" name="Main_wd">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>180</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>SIMPLE stats analyzer</string>
  </property>
  <widget class="QPushButton" name="btn_path_finder">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>34</y>
     <width>75</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
    </font>
   </property>
   <property name="text">
    <string>Find path</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupbox_1">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>64</y>
     <width>300</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string>Options</string>
   </property>
   <widget class="QRadioButton" name="radiobtn_both">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
     </font>
    </property>
    <property name="text">
     <string>Show picture + save</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radiobtn_justsave">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>90</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
     </font>
    </property>
    <property name="text">
     <string>Just save</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_dragdrop">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>14</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>↓ Drag &amp; Drop</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_stats_analysis">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>72</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
    </font>
   </property>
   <property name="text">
    <string>Stats analysis</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_dragdrop_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>111</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>↓ save folder</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_savepath_open">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>131</y>
     <width>75</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
    </font>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_find_path">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>34</y>
     <width>650</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_save_path">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>131</y>
     <width>650</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
